doc/api/path.markdown

```diff
@@ -117,6 +119,20 @@ path.format({
 '/file.txt'
 ```
 
+An example on Windows:
+
+```js
+path.format({
+    root : "C:\\",
+    dir : "C:\\path\\dir",
+    base : "file.txt",
+    ext : ".txt",
+    name : "file"
+})
+// returns
+'C:\\path\\dir\\file.txt'
```

Comment:
@mithun-daa I would say no, or at least not until #5670 lands. It might now land. I think what you've done here is fine.